Docs/Tutorials/Waypoint/Software/Apps/TouchGFX-GUI/una/Makefile

```
@@ -0,0 +1,276 @@
+fontconvert_font_files := $(fontconvert_ttf_lower_files) \
+			  $(fontconvert_ttf_upper_files) \
+			  $(fontconvert_otf_lower_files) \
+			  $(fontconvert_otf_upper_files) \
+			  $(fontconvert_bdf_lower_files) \
+			  $(fontconvert_bdf_upper_files)
+
+source_files := $(call find, $(source_paths),*.cpp) $(framework_files) $(ADDITIONAL_SOURCES)
+c_source_files := $(call find, $(source_paths),*.c)
+
+object_files := $(source_files:$(touchgfx_path)/%.cpp=$(object_output_path)/touchgfx/%.o) 
+$(c_source_files:$(touchgfx_path)/%.c=$(object_output_path)/touchgfx/%.o)
```

checkmake (0.3.2)

[warning] Target "$(c_source_files" should be declared PHONY.

(phonydeclared)